src/operations/query/filters/id.js

```diff
@@ -66,6 +66,84 @@ class FilterById extends BaseFilter {
         return query;
 
     }
+
+    /**
+     * Generates filter for parsedArgItem
+     * @return {import('mongodb').Filter<import('mongodb').DefaultSchema>[]}
+     */
+    filter() {
```

Comment:
See if we can replace the usage of getListFilter() above with this function since getListFilter() is also used to handle the IN query.

Reply:
Modified the function to use existing functions so that code is reused